docs/tutorial/01-getting-started.md

```diff
@@ -0,0 +1,357 @@
+    }
+}
+```
+
+Finally, we add the `guess` function which accepts a number as the guess and compares it to the stored number, returning the result. Notice we're using our defined key (that small Symbol) to find stored data that may or may not be there. That's why we need [`unwrap()`](https://doc.rust-lang.org/rust-by-example/error/option_unwrap.html), but we'll talk more about `Option` values later in the tutorial.
+
+```rust
+mod test;
+```
+
+Post Script: this last line includes the test module into this file. It's handy to write unit tests for our code in a separate file (`contracts/guess-the-number/src/test.rs`), but you could also write them inline if you want.
+
+
+## 👷‍♀️ Let's Build and Test It Locally
+
+Now that we have some contract code in Rust, we can deploy it and try interacting with it. Eventually we'll run this contract on Stellar's Mainnet, the production level network with real users and financial connections. But we should test our code first. Stellar maintains a Testnet, a free-to-use network that mimics a production environment. That's a perfect place to test with real network conditions and for sharing test code with others, and we'll do this later. But there's an even quicker option: running a local network.
+
+> ℹ️ [Read more about the differences between networks.](https://developers.stellar.org/docs/learn/fundamentals/networks)
+
+Stellar CLI has a built-in command to easily configure and manage a Docker container that will run the local network for you. Make sure Docker is running in the background and run the following command in your terminal:
+
+```bash
+$ stellar container start local
+  ℹ️ Starting local network
+  ...
+  ✅ Started container
+```
+
+This will download the image and start up the container as a mini Stellar network ready for us to use. Now let's build the contract. That means compiling the Rust code into WebAssembly (a `.wasm` file that can run on the network) as well as [creating the interface types](https://developers.stellar.org/docs/learn/fundamentals/contract-development/types/fully-typed-contracts) (a specification that can be used by other contracts or code to interact with your contract). 
+
+```bash
+$ stellar contract build
+```
+
+Sharp eyed readers might notice these lines in the build output:
+
+```
+  Exported Functions: 7 found
+    • _
+    • __constructor
+    • admin
+    • guess
+    • reset
+    • set_admin
+    • upgrade
+✅ Build Complete
+```
+
+We implemented a few of those functions in our contract (i.e. `__constructor`, `guess`, and `reset`), but where did the others come from? They are inherited from the `Administratable` and `Ugradable` traits we talked about earlier.
+
+## 🚢 Deploy To Local Network
+
+In order to deploy the contract, we need an account on the local network to own the contract and sign transactions. Let's do that now and also provide some [fake funds](https://developers.stellar.org/docs/learn/fundamentals/networks#friendbot) to test with.
```

Comment:
This should use scaffold stellar tooling. Scaffold stellar build should have made the keys for the local network.